ompi/mca/coll/libnbc/nbc_ialltoallv.c

```diff
@@ -74,6 +74,11 @@ int ompi_coll_libnbc_ialltoallv(const void* sendbuf, const int *sendcounts, cons
       }
     }
     span = opal_datatype_span(&recvtype->super, count, &gap);
+    if (OPAL_UNLIKELY(0 == span)) {
```

Comment:
Right, but #3719 is for master, and I am not sure it will be back-ported to the 2.x. My comment doesn't point to a real issue, mostly to a missed optimization opportunity. Once you add 68ac950 I will review it again.